spec/support/shared_examples/sidebar_menu_activatable.rb

```diff
@@ -0,0 +1,5 @@
+shared_examples_for "activatable" do |activated|
+  it 'is actived now' do
+    expect(page).to have_css('.active', activated)
```

Comment:
@Gataniel also let's add within here, so we will search only in some specific area, not in whole page.